app/components/users/library/library-entry.js

```diff
@@ -17,6 +17,20 @@ export default Component.extend({
     return get(this, 'media.unitCount') || '-';
   }).readOnly(),
 
+  /**
+   * Returns keys used in our translation file.
+   */
+  airingStatus: computed('media.{startDate,endDate,unitCount}', function() {
```

Comment:
@vevix is there a place we can put this method so that both `media-information` and `library-entry` components can share it?